examples/pom.xml

```diff
@@ -36,4 +33,14 @@
         <module>custom-mailets</module>
     </modules>
 
-</project>
\ No newline at end of file
+    <properties>
+        <james.groupId>org.apache.james</james.groupId>
+        <james.baseVersion>3.6.0-SNAPSHOT</james.baseVersion>
```

Comment:
I understand the concerns however it likely makes the release process hard.

How do we ensure the version use by the example matches the released version? How does it integrate with the maven-release plugin?

I I was to tag a 3.6.0 tomorrow, I would likely forget that. Even if I don't forget, how do I ensure the released exemple JAR matches the 3.6.0 version? Would I have to do a pre-release commit switching it to the not yet released 3.6.0 - in which case the whole process might fail at publishing snapshots?

Maybe a wiser value here would be :

```
<james.baseVersion>${project.version}</james.baseVersion>
```

That way as the version of the example and James are implicitly aliagned, the maven-release prompt will enable me to do the switch painlessly...

Reply:
Thanks for good feedback. I didn't release the examples are get published so of course it makes sense to use this property as a default.